integration-tests/rest-client/pom.xml

```diff
@@ -1,132 +1,134 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0"
-	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
Can you please restore the formatting on this file? The change makes it hard to review

Reply:
Done.

Reply:
@geoand For info it's possible to ignore white-spaces noise in diffs: https://stackoverflow.com/questions/37007300/how-to-ignore-whitespace-in-github-when-comparing

Reply:
Didn't know that one, thanks! `w=1` isn't that easy to figure out 😂